lib/ui/navbar.js

```diff
@@ -363,19 +372,46 @@ export default class IpNavBar extends React.Component {
         "connecting": true,
       });
       SignalkClient.on('connect', () => {
-        SignalkClient.subscribe({context:'vessels.self',subscribe:[{ path:'*'}]});
-        SignalkClient.on('delta', this.props.instrumentPanel.dispatch.bind(this.props.instrumentPanel));
+        // For debug only with npm start
```

Comment:
Why the token authentication here?

Reply:
I need it for easy debug with `npm start` when IP running at 3001 listening port